examples/MNIST/README.md

```diff
@@ -1,4 +1,10 @@
-# Deploying MNIST Handwritten Recognition using torchserve
+# Deploying MNIST Handwritten Recognition using TorchServe
+
+## Training the model
+
+Follow the [respository](https://github.com/mlflow/mlflow/tree/master/examples/pytorch/MNIST/example1) to train MNIST handwritten digit recognition.
```

Comment:
@karthik-77  it seems, the parameters in the MLProject file, mnist_mode.py and the command from the referenced repository needs to be synced. For example the command is using batch_size where in the  mnist_mode.py, it is been referred as batch-size and also no present in the MLProject.

Reply:
Looks like a old edited file. We have a independent [model file](https://github.com/mlflow/mlflow-torchserve/blob/master/examples/MNIST/mnist_model.py) for this example. I have reverted the changes in the read me file, to be in sync with the master. The MLProject files also seems to be in sync.